applications/external/picopass/picopass_device.h

```diff
@@ -24,6 +25,49 @@
 #define PICOPASS_AIA_BLOCK_INDEX 5
 #define PICOPASS_PACS_CFG_BLOCK_INDEX 6
 
+// Personalization Mode
+#define PICOPASS_FUSE_PERS 0x80
+// Crypt1 // 1+1 (crypt1+crypt0) means secured and keys changable
+#define PICOPASS_FUSE_CRYPT1 0x10
+// Crypt0 // 1+0 means secure and keys locked, 0+1 means not secured, 0+0 means disable auth entirely
+#define PICOPASS_FUSE_CRTPT0 0x08
+#define PICOPASS_FUSE_CRYPT10 (PICOPASS_FUSE_CRYPT1 | PICOPASS_FUSE_CRTPT0)
+// Read Access, 1 meanns anonymous read enabled, 0 means must auth to read applicaion
+#define PICOPASS_FUSE_RA 0x01
+
+// PicoPass command bytes:
+// Low nibble used for command
+// High nibble used for options and checksum (MSB)
+// The only option we care about in 15693 mode is the key
+// which is only used by READCHECK, so for simplicity we
+// don't bother breaking down the command and flags into parts
+// READ: ADDRESS(1) CRC16(2) -> DATA(8) CRC16(2)
+// IDENTIFY: No args -> ASNB(8) CRC16(2)
+#define PICOPASS_CMD_READ_OR_IDENTIFY 0x0C
```

Comment:
I think this is the same as `RFAL_PICOPASS_CMD_READ` in `rfal_picopass.h`, except yours has a better explanation.  If they are the same, feels like we shouldn't duplicate, although I'm up in the air as to the 'better' file to put them in.  What do you think?

Reply:
Herp I missed the ones in rfal_picopass.h. Agreed it's worth collapsing them into one. Will fix this up soon.

Reply:
Merged them together, also cleaned up some other duplicated defined (noticed the block size defined in two different places as well)